compiler/ballerina-lang/src/main/resources/compiler.properties

```diff
@@ -1690,3 +1690,6 @@ error.invalid.readonly.field.type=\
 error.continue.not.allowed=continue not allowed here
 
 error.break.not.allowed=break not allowed here
+
+error.method.call.expr.not.allowed=\
+  method call expression not allowed on field ''{0}'': expected a function type, but found ''{1}''
```

Comment:
Shouldn't the error be something like

```
invalid method call expression: expected a function type, found ''{0}''
```

When we say "on field", it kinda feels like we are trying to access the field and then call a method on it, but we get this error when we are actually trying to call the field itself?